easybuild/tools/github.py

```diff
@@ -1271,6 +1271,41 @@ def merge_url(gh):
         print_warning("Review indicates this PR should not be merged (use -f/--force to do so anyway)")
 
 
+@only_if_module_is_available('git', pkgname='GitPython')
+def post_pr_labels(pr, labels):
+    """
+    Update PR labels
+    """
+    pr_target_account = build_option('pr_target_account')
+    pr_target_repo = build_option('pr_target_repo')
+
+    # fetch GitHub token (required to perform actions on GitHub)
+    github_user = build_option('github_user')
+    if github_user is None:
+        raise EasyBuildError("GitHub user must be specified to open a pull request")
+
+    github_token = fetch_github_token(github_user)
+    if github_token is None:
+        raise EasyBuildError("GitHub token for user '%s' must be available to open a pull request", github_user)
```

Comment:
Is an error really the right thing here?
Anyone can do review-pr, they don't need an active token for that.
If it's a review only then it should just return False and not error out.

And we're not necessarily opening a pull request, we're might be reviewing one...